@here/harp-examples/src/styling_data-driven.ts

```diff
@@ -93,7 +93,7 @@ export namespace DataDrivenThemeExample {
     function main() {
         const map = initializeMapView("mapCanvas");
 
-        const omvDataSource = new OmvDataSource({
+        const baseMapData = new VectorTileDataSource({
```

Comment:
For consistency in examples should use either "baseMapData" or "mapData" everywhere.